test/support/helpers/context_dir.js

```diff
@@ -122,6 +122,10 @@ class ContextDir {
     return fs.outputFile(this.abspath(target), data)
   }
 
+  async move (src /*: string */, dst /*: string */) {
+    return fs.move(this.abspath(src), this.abspath(dst))
```

Comment:
Doesn't `fs-extra`'s `#move()` do quite a lot of magic under the hood compared to plain `rename` call?
Could it be an issue?

Reply:
It does, but it's not an issue. ContextDir also uses `fs-extra` on other operations that can have some magic under the hood.